tests/bsim/compile.source

```diff
@@ -19,6 +19,7 @@ function _compile(){
   local app_root="${app_root:-${ZEPHYR_BASE}}"
   local BOARD_ROOT="${BOARD_ROOT:-${ZEPHYR_BASE}}"
   local conf_file="${conf_file:-prj.conf}"
+  local extra_conf_file="${extra_conf_file:-""}"
```

Comment:
Nit: It would be nice to separate this file change in its own commit

Reply:
done